docs/source/user_guide/installation/index.rst

```diff
@@ -1,6 +1,14 @@
+
+
+Required dependencies
+=====================
+
+PyBaMM requires the following dependencies.
+
```

Comment:
This seems out of place. This is the main installation page for PyBaMM, not a page for required dependencies

Reply:
sorry about that